src/Cake.Testing/FakeEnvironment.cs

```diff
@@ -52,6 +55,10 @@ public FakeEnvironment(PlatformFamily family, bool is64Bit = true)
         {
             Platform = new FakePlatform(family, is64Bit);
             Runtime = new FakeRuntime();
+
+            var launchInfo = new FakeLaunchInfo();
+            LaunchDirectory = launchInfo.LaunchDirectory;
```

Comment:
I believe you'll need to expose `FakeLaunchInfo` in this `FakeEnvironment` in order to manipulate it during your unit tests (as per your previous comment on following the pattern of Runtime). Sorry if I caused any confusion with my question about why you needed the two implementations - it makes sense to expose the `Fake` to your tests

Reply:
I'm not sure how I can have two implementations because we don't have the `ILaunchInfo` property anymore in `ICakeEnvironment`